contracts/prize-pool/PrizePool.sol

```diff
@@ -862,8 +866,14 @@ abstract contract PrizePool is PrizePoolInterface, OwnableUpgradeable, Reentranc
   /// @dev Checks if a specific token is controlled by the Prize Pool
   /// @param controlledToken The address of the token to check
   /// @return True if the token is a controlled token, false otherwise
-  function _isControlled(address controlledToken) internal view returns (bool) {
-    return _tokens.contains(controlledToken);
+  function _isControlled(ControlledTokenInterface controlledToken) internal view returns (bool) {
```

Comment:
Ah!  While you're here: let's add a public accessor!

Reply:
Sure!